src/forms/forms.response.ts

```diff
@@ -102,6 +117,21 @@ class BaseQuestion {
     @ApiProperty({ example: null })
     optionGroup: OptionGroup;
 
+    @Optional()
+    @ApiProperty({
+        examples: {
+            scaleConfig: {
```

Comment:
would it be easier/better to just keep it `parseConfig`? otherwise we'll have to add a new `xConfig` for every control, there could be 20+

I also remember you had parseConfig for everything in the last PR, is there any reason to change it to "multiple config"